### ISSUE TITLE
AutoHotkey and installation errors "Error: Parameter #3 must not be blank in this case"

### ISSUE THREAD

**Issue by PhilippS23** [rbreaves/kinto#733]
I tried to reinstall Kinto multiple times, but I just can't install it correctly. Latest versions of Python and Git are installed. But I still get this error after running the script.
![grafik](https://user-images.githubusercontent.com/76483538/179758554-a27a26d3-2e38-4590-804a-122c59692313.png)

By running the command "py .\setup.py" there is also an error at the installation progress. The output is in German, but it says "The system cannot find the specified path"

![grafik](https://user-images.githubusercontent.com/76483538/179759407-c3e0681a-5da2-4eb0-871a-c3842a991c9c.png)

If I ignore the error and just press "y" it will install the application.

At the end of the installation, there is also the same error as in the second photo.

![grafik](https://user-images.githubusercontent.com/76483538/179763333-90fc01ee-55c0-4926-8f0d-c9ac9e3d0164.png)

The Device I am using is a Lenovo ThinkPad X1 Yoga Gen 6 applied to a docking station with a normal office keyboard.

If I restart the device, I directly get the error of the first photo after logging in.

Does anyone have an idea how to fix this?

**Comment by rbreaves** [comment]
Blank parameters usually indicate that Strawberry Perl is either not installed, installed correctly or not being called properly.. I am really not sure which is going on, that log indicates you have Strawberry Perl. Only thing I know that could cause that error though.

**Comment by hieusydo** [comment]
@PhilippS23 FYI, I ran into this exact same error when updating kinto. Eventually, I had to _**completely**_ remove everything and reran the setup script. Now it's working again. 
1. `py .\setup.py` and choose option 5 for uninstall. 
2. Uninstall Strawberry & chocolatey. This varies depending how you installed them at first. But make sure that `C:\Strawberry` and `C:\ProgramData\chocolatey` are _removed_ before try reinstalling. 
3. Reran setup script.

**Comment by bigplayer-ai** [comment]
hey did you find any solution? this happens after choosing apple keyboard, then switching back to windows keyboard... or restarting kinto
```
---------------------------
kinto.ahk
---------------------------
Error:  Parameter #3 must not be blank in this case.

	Line#
	047: checkKB = Apple
	048: if paramkb = win
	049: checkKB = Windows
	050: if paramkb = chrome
	051: checkKB = Chromebook
	052: if paramkb = ibm
	053: checkKB = IBM (No Super/Win key)
--->	055: Menu,Keyboards,check,%checkKB%
	059: Menu,Tray,NoStandard
	060: Menu,Tray,Add,Keyboard Types,:Keyboards
	061: Menu,Tray,Add,Autodetect Keyboards,autodetect
	064: Menu,Tray,Add,Suspend Kinto,tray_suspend
	066: Menu,Tray,Add,OptSpecialChars   Shift+Opt+Cmd+O,toggle_optspecialchars
	068: Menu,Tray,Add
	069: Menu,Tray,Add,Close,Exit

The current thread will exit.
---------------------------
OK   
---------------------------
```

**Comment by PhilippS23** [comment]
> hey did you find any solution?

Nope, sadly not. My colleague just agreed to work with the regular Windows shortcuts.
